RNS/Utilities/rnodeconf.py

```diff
@@ -3440,6 +3440,10 @@ def get_flasher_call(platform, fw_filename):
                                 time.sleep(6.5)
 
                             elif rnode.platform == ROM.PLATFORM_NRF52:
+                                # Wait a few seconds before hard resetting.
+                                # Otherwise, macOS fails to set firmware hash on NRF52
+                                time.sleep(5)
```

Comment:
If this _only_ happens on macOS, could you wrap it in a `RNS.vendor.platformutils.is_darwin()` check as well?

Reply:
Cannot confirm if this is 100% limited to macOS, I haven't been able to test on Windows yet. However, it was confirmed working on Linux without this being added.

Reply:
Ok, I suggest limiting it to darwin only for now then.